src/Engine/ProtoCore/Reflection/MirrorData.cs

```diff
@@ -21,6 +21,14 @@ public class MirrorData
             /// </summary>
             private StackValue svData;
 
+            public bool IsFunction
+            {
+                get
+                {
+                    return svData.IsPointer && Class.Name.Equals("Function") && Class.LibraryMirror.Name.Equals("FunctionObject.ds");
```

Comment:
Do we have constants already defined for "Function" and "FunctionObject.ds"? Can you search for these in the codebase? If not, I think we should create them in a common utility class and make them accessible in the `ProtoCore` project at least.

Reply:
I didn't find any constants defined for that. Do you have a class in mind for that?

Reply:
`ProtoCore.Utils.CoreUtils` maybe

Reply:
Ok, done. Thanks!